hstream/hstream.cabal

```diff
@@ -73,6 +73,8 @@ test-suite hstream-test
   build-depends:
     , aeson
     , base                >=4.11 && <5
+    , bytestring
+    , clickhouse-haskell
```

Comment:
why add this ?

Reply:
Because in the RunSQLSpec.hs we need clickhouse-haskell and bytestring.